crates/networking/rpc/net/mod.rs

```diff
@@ -0,0 +1,12 @@
+use ethereum_rust_storage::Store;
+use serde_json::Value;
+
+use crate::utils::{RpcErr, RpcRequest};
+
+pub fn version(_req: &RpcRequest, store: Store) -> Result<Value, RpcErr> {
+    let chain_spec = store
+        .get_chain_config()
+        .map_err(|error| RpcErr::Internal(error.to_string()))?;
```

Comment:
`RpcErr` implements `From<StoreError>` so we don't need to explicitly map it and can use `?` directly instead

Reply:
ok fixed

Reply:
btw, this explicit error mapping of `StoraError` is being done also in other rpc methods that fetch `chain_spec`. Should I refactor all such occurences or just leave it for now?